app/models/pusher.rb

```diff
@@ -43,7 +43,14 @@ def save
   end
 
   def pull_spec
-    @spec = Gem::Package.new(body).spec
+    # Verify that the gem signatures match the certificate chain (if present)
+    policy = Gem::Security::LowSecurity.dup
+    # Silence warnings from the verification
+    stream = StringIO.new
+    policy.ui = Gem::StreamUI.new(stream, stream, stream, false)
+
+    package = Gem::Package.new(body, policy)
```

Comment:
Hi, can we please move this policy setup to a private function. following would look cleaner:
```
package = Gem::Package.new(body, low_security_policy)
```